README.md

```diff
@@ -90,32 +90,33 @@ One of the primary advantage of CMake based build system is integration with oth
   pushd build
   ```
 
-3. Run cmake with the appropriate options:
+3. Run cmake with the appropriate options, e.g.:
 
   ```
-  cmake \
+  cmake .. \
    -DNRN_ENABLE_PYTHON=ON \
```

Comment:
Wonder if this should be eliminated as it is the default. The rest are good as they are the opposites of the default.